frontend/views/components/TextWho.vue

```diff
@@ -1,21 +1,17 @@
-<template>
-  <span>
-    {{firstWho}}
-    <template v-if="hasWhoElse">
-      <i18n>and</i18n>
-      <tooltip>
-        <button type="button" class="gi-is-unstyled gi-is-link-inherit">{{notFirstWho.length}}<i18n>others</i18n></button>
-        <template slot="tooltip">
-          <p v-for="(name, index) in notFirstWho" :key="`name-${index}`">{{name}}</p>
-        </template>
-      </tooltip>
-    </template>
-  </span>
+<template lang="pug">
+span
+  | {{firstWho}}
+  template(v-if='hasWhoElse')
+    i18n and
+    tooltip
+      button.is-unstyled.is-link-inherit(type='button')
+        | {{notFirstWho.length}}
+        i18n others
```

Comment:
Can you add a comment here noting that the `button` tag should be made an `i18n` tag, e.g. `<i18n tag="button">`. This will make it easier to translate the string in context.

I.e. it's a bad idea to translate the individual word "others", it should translated in context.

So that it looks like:

```html
i18n(tag='button' :args='{numOthers: notFirstWho.length}') {numOthers} others
```

This way the translator sees `{numOthers} others` instead of just `others`. Then they can move the position of `{numOthers}` if necessary.